### ISSUE TITLE
Add Signup Form to Enable User Registration

### ISSUE THREAD

**Issue by pragna7** [PratikMane0112/TelMedSphere#23]
### Description

The project currently includes a login form for existing users but lacks a signup form to allow new users to register and create accounts. This limitation restricts user accessibility and prevents potential users from exploring the platform or its features.

### Screenshots

![Screenshot 2024-12-27 220850](https://github.com/user-attachments/assets/d6e6a370-0a9d-482a-9862-faa7ca204454)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a SWOC'25 contributor.
- [ ] I am a KWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/PratikMane0112/TelMedSphere/blob/master/.github/CODE_OF_CONDUCT.md)".

**Comment by PratikMane0112** [comment]
![Screenshot 2024-12-28 021431](https://github.com/user-attachments/assets/00bebf57-40ef-4c84-b123-7c1f460ee8f9)

Hey @pragna7 , Welcome to 💖TelMedSphere!!!!! Thanks for showing your interest but I think you get this issue in wrong way. There is sigup form in website, you can see it just clicking on "create an account". Feel free to create any other issue, you can take reference of below -
https://medium.com/@pratikmane610/beginner-friendly-issues-for-contributing-to-open-source-062e0f16ceab
